config/systems.php

```diff
@@ -47,15 +47,15 @@
         'active' => true,
     ],
     7 => [
-        'name' => 'NES',
-        'name_full' => 'Nintendo Entertainment System',
+        'name' => 'NES/Famicom',
+        'name_full' => 'Nintendo Entertainment System/Famicom',
         'name_short' => 'NES',
         'manufacturer' => 'Nintendo',
         'active' => true,
     ],
     8 => [
-        'name' => 'PC Engine / TurboGrafx',
-        'name_full' => 'NEC PC Engine / TurboGrafx',
+        'name' => 'PC Engine/TurboGrafx16',
```

Comment:
There should be a hyphen in `TurboGrafx-16`: https://en.wikipedia.org/wiki/TurboGrafx-16

Reply:
good catch: [TurboGrafx-16](https://github.com/RetroAchievements/RAWeb/pull/2172/commits/cdd2d9f383eae7b445b317e8503764b1ad86845e)